src/plugins/template/tests/functional/op_reference/comparison.hpp

```diff
@@ -5,15 +5,35 @@
 #include <gtest/gtest.h>
 
 #include "base_reference_test.hpp"
-#include "ngraph_functions/builders.hpp"
+#include "openvino/op/equal.hpp"
+#include "openvino/op/not_equal.hpp"
+#include "openvino/op/greater.hpp"
+#include "openvino/op/greater_eq.hpp"
+#include "openvino/op/less.hpp"
+#include "openvino/op/less_eq.hpp"
+#include "openvino/op/is_inf.hpp"
+#include "openvino/op/is_finite.hpp"
+#include "openvino/op/is_nan.hpp"
 
 using namespace ov;
 
 namespace reference_tests {
 namespace ComparisonOpsRefTestDefinitions {
 
+enum ComparisonTypes {
```

Comment:
As I see this enum is used in different places, should we move it to new test utils?

Reply:
done